File: CiscoPCCE.Toolkit/Bean/LicenseReservationRequest.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlType("licenseReservation")]
    public class LicenseReservationRequest : BaseApiBean
    {
        [XmlElement("reservationCode")]
        public string? ReservationCode { get; set; }
    }
}

File: JavaSrc/toolkit/bean/LicenseReservationRequest.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="licenseReservation")
public class LicenseReservationRequest extends BaseApiBean {
  private String reservationCode;

  public String getReservationCode() {
     return this.reservationCode;
  }

  public void setReservationCode(String reservationCode) {
     this.reservationCode = reservationCode;
  }


}
